livingdoc-engine/src/main/kotlin/org/livingdoc/engine/execution/examples/ExampleResult.kt

```diff
@@ -0,0 +1,4 @@
+package org.livingdoc.engine.execution.examples
+
+interface ExampleResult {
```

Comment:
Can we get a different name for this, like ElementResult, or something else? `Example` always makes me think this is some kind of showcase code.